### ISSUE TITLE
Update mysql db variables in recap playbooks

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/princeton_ansible#3118]
Still pointing at old server maria-staging2.princeton.edu, needs to be mysql-db-staging1. Playbook fail to run. 

```
atal: [recap-www-staging1.princeton.edu]: FAILED! => {"changed": false, "cmd": ["/home/deploy/sql/set_permission.sh"], "delta": "0:00:18.564511", "end": "2022-06-24 02:16:04.843406", "msg": "non-zero return code", "rc": 1, "start": "2022-06-24 02:15:46.278895", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.\nERROR 2003 (HY000): Can't connect to MySQL server on 'maria-staging2.princeton.edu' (113)", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure.", "ERROR 2003 (HY000): Can't connect to MySQL server on 'maria-staging2.princeton.edu' (113)"], "stdout": "", "stdout_lines": []}```

**Comment by christinach** [comment]
Use this branch https://github.com/pulibrary/princeton_ansible/pull/3091/files until we do a database upgrade of production to maria 10.3 https://github.com/pulibrary/princeton_ansible/pull/3119#issuecomment-1165623899
